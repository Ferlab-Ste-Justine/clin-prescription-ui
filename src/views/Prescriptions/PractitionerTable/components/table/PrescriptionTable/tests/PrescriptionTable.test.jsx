import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store';

import PrescriptionTable from '../index';

describe.skip('PrescriptionTable', () => {
  test.skip('should be as the snapshot', () => {
    const initialState = { user: { user: { config: {} } } };
    const store = configureStore()(initialState);
    const component = renderer.create(
      <Provider store={store}>
        <PrescriptionTable queryConfig={{}} />
      </Provider>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
