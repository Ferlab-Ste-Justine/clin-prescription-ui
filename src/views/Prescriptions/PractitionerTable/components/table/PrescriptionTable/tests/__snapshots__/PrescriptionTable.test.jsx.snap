// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrescriptionTable should be as the snapshot 1`] = `
<div
  className="ant-space ant-space-vertical"
  style={Object {}}
>
  <div
    className="ant-space-item"
    style={Object {}}
  >
    <div>
      <div
        className="ant-space ant-space-horizontal ant-space-align-center"
        style={Object {}}
      >
        <div
          className="ant-space-item"
          style={Object {}}
        >
          <span
            className="ant-typography"
            onClick={null}
            style={
              Object {
                "WebkitLineClamp": undefined,
              }
            }
          >
            Aucun r√©sultat
          </span>
        </div>
      </div>
      <div
        className="ant-space ant-space-horizontal ant-space-align-center"
        style={Object {}}
      >
        <div
          className="ant-space-item"
          style={Object {}}
        >
          <div>
            <button
              className="ant-btn ant-btn-text ant-btn-sm ant-btn-icon-only"
              disabled={false}
              onClick={[Function]}
              onMouseEnter={[Function]}
              onMouseLeave={[Function]}
              type="button"
            >
              <span
                aria-label="download"
                className="anticon anticon-download"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  data-icon="download"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M505.7 661a8 8 0 0012.6 0l112-141.7c4.1-5.2.4-12.9-6.3-12.9h-74.1V168c0-4.4-3.6-8-8-8h-60c-4.4 0-8 3.6-8 8v338.3H400c-6.7 0-10.4 7.7-6.3 12.9l112 141.8zM878 626h-60c-4.4 0-8 3.6-8 8v154H214V634c0-4.4-3.6-8-8-8h-60c-4.4 0-8 3.6-8 8v198c0 17.7 14.3 32 32 32h684c17.7 0 32-14.3 32-32V634c0-4.4-3.6-8-8-8z"
                  />
                </svg>
              </span>
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
